src/pages/Host/LiveSession/HostLiveSession.css

```
@@ -1,41 +1,655 @@
+.host-live-wave-bar {
+  width: 4px;
+  border-radius: 4px;
+  background: linear-gradient(to top, #1a9fd4, #55c5f1);
+  animation: waveBounce 1s ease-in-out infinite;
+}
+.host-live-wave-bar:nth-child(1) { animation-delay: 0s; height: 12px; }
+.host-live-wave-bar:nth-child(2) { animation-delay: 0.15s; height: 24px; }
+.host-live-wave-bar:nth-child(3) { animation-delay: 0.3s; height: 18px; }
+.host-live-wave-bar:nth-child(4) { animation-delay: 0.1s; height: 30px; }
+.host-live-wave-bar:nth-child(5) { animation-delay: 0.25s; height: 14px; }
+@keyframes waveBounce {
```

Stylelint (17.7.0)

[error] Expected keyframe name "waveBounce" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)